src/migrator.js

```diff
@@ -9,8 +9,9 @@ export default class extends Command {
   plugins: Plugins
 
   async run () {
+    if (fs.existsSync(path.join(this.config.dirname, 'plugins', 'package.json'))) return
```

Comment:
Joining the file path this way instead of using `this.plugins.userPluginsPJSONPath` to save the millisecs from instantiating a new plugins object.